safe/impact_functions/volcanic/volcano_polygon_building/impact_function.py

```diff
@@ -46,6 +46,9 @@ def __init__(self):
         self.volcano_names = tr('Not specified in data')
         self._target_field = 'Hazard'
 
+        # From BuildingExposureReportMixin
+        self.building_report_threshold = 25
```

Comment:
Yes, it will be replaced by the value from the postprocessor, but don't we need to initialise every var which are in ```self``` in the ```__init__``` of the current class ?